src/main/java/swervelib/SwerveDrive.java

```diff
@@ -1046,7 +1046,7 @@ public void addVisionMeasurement(Pose2d robotPose, double timestamp) {
         robotPose.getRotation());
     odometryLock.unlock();
 
-    setGyroOffset(new Rotation3d(0, 0, robotPose.getRotation().getRadians()));
+    //setGyroOffset(new Rotation3d(0, 0, robotPose.getRotation().getRadians()));
```

Comment:
is this intended to be used?

Reply:
Yeah, we are just disregrading rotation data from vision and using the NavX data because it is more accurate.